README.md

```diff
@@ -1,2 +1,4 @@
 # Learning Python
 My scripts for learning Python
+
+test.py description
```

Comment:
Please, provide a detailed description and rename file from **test.py** to **alarm-solution.py**